client/context/context.jsx

```diff
@@ -5,13 +5,34 @@ import axios from 'axios';
 export const AppContext = createContext();
 
 export const AppProvider = ({ children }) => {
+<<<<<<< HEAD
```

Comment:
Please resolve this conflict. You may not have pulled the latest version of the code